govuk_personalisation.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = %w[lib]
 
   spec.add_dependency "plek", ">= 1.9.0"
-  spec.add_dependency "rails", "~> 6"
+  spec.add_dependency "rails", "~> 7"
```

Comment:
Can we make this `>= 6 && < 8` (or whatever the appropriate gemspec syntax is)?  As it is, this is a breaking change because you're removing compatibility with rails 6.

Reply:
Actually, we probably don't want to run our tests against rails 6 and rails 7, which we should ideally do if we're claiming compatibility with both.  So let's just go for rails 7.